docs/apis/http/webdav/_index.md

```diff
@@ -0,0 +1,533 @@
+                <d:getcontentlength>5</d:getcontentlength>
+                <oc:size>5</oc:size>
+                <d:getlastmodified>Mon, 28 Aug 2023 20:45:03 GMT</d:getlastmodified>
+                <d:getetag>"75115347c74701a3be9c635ddebbf5c4"</d:getetag>
+                <d:getcontenttype>text/plain</d:getcontenttype>
+                <d:resourcetype></d:resourcetype>
+            </d:prop>
+            <d:status>HTTP/1.1 200 OK</d:status>
+        </d:propstat>
+    </d:response>
+    <d:response>
+        <d:href>/remote.php/dav/spaces/storage-users-1$some-admin-user-id-0000-000000000000/NewFolder/</d:href>
+        <d:propstat>
+            <d:prop>
+                <oc:permissions>RDNVCKZP</oc:permissions>
+                <oc:favorite>0</oc:favorite>
+                <oc:fileid>storage-users-1$some-admin-user-id-0000-000000000000!5c73ecd9-d9f4-44f4-b685-ca4cb40aa6b7</oc:fileid>
+                <oc:id>storage-users-1$some-admin-user-id-0000-000000000000!5c73ecd9-d9f4-44f4-b685-ca4cb40aa6b7</oc:id>
+                <oc:owner-id>admin</oc:owner-id>
+                <oc:owner-display-name>Admin</oc:owner-display-name>
+                <oc:privatelink>https://localhost:9200/f/storage-users-1$some-admin-user-id-0000-000000000000%215c73ecd9-d9f4-44f4-b685-ca4cb40aa6b7</oc:privatelink>
+                <oc:size>0</oc:size>
+                <d:getlastmodified>Mon, 28 Aug 2023 20:45:10 GMT</d:getlastmodified>
+                <d:getetag>"e83367534cc595a45d706857fa5f03d8"</d:getetag>
+                <d:resourcetype>
+                    <d:collection/>
+                </d:resourcetype>
+            </d:prop>
+            <d:status>HTTP/1.1 200 OK</d:status>
+        </d:propstat>
+        <d:propstat>
+            <d:prop>
+                <oc:checksums></oc:checksums>
+                <oc:share-types></oc:share-types>
+                <d:getcontentlength></d:getcontentlength>
+                <d:getcontenttype></d:getcontenttype>
+            </d:prop>
+            <d:status>HTTP/1.1 404 Not Found</d:status>
+        </d:propstat>
+    </d:response>
+</d:multistatus>
+```
+{{< /tab >}}
+{{< tab "400 - Bad Request" >}}
+
+#### Body
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<d:error xmlns:d="DAV" xmlns:s="http://sabredav.org/ns">
+    <s:exception>Sabre\DAV\Exception\BadRequest</s:exception>
+    <s:message>Invalid Depth header value: 3</s:message>
+</d:error>
+```
+
+This can occur if the request is malformed e.g. due to an invalid xml request body or an invalid depth header value.
+{{< /tab >}}
+{{< tab "404 - Not Found" >}}
+
+#### Body
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<d:error xmlns:d="DAV" xmlns:s="http://sabredav.org/ns">
+    <s:exception>Sabre\DAV\Exception\NotFound</s:exception>
+    <s:message>Resource not found</s:message>
+</d:error>
+```
+{{< /tab >}}
+{{< /tabs >}}
+
+### Request Body
```

Comment:
The headers here just say "Body" - didn't you want to make them more specific?

Reply:
It is in the tabs view. I want to always have `Body` and `Headers`